app/WikiEntitiesCount.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class WikiEntitiesCount extends Model
+{
+    use HasFactory;
+
+    const FIELDS = [
+        'items_count',
+        'properties_count',
+        'wiki_id'
+    ];
+
+    protected $fillable = self::FIELDS;
+
+    protected $visible = self::FIELDS;
+
+    protected $casts = [
```

Comment:
Question: seeing these fields are declared as integers in the database schema already, is the cast still needed?